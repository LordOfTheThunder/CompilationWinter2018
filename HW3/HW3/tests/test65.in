bool RetBool() {
	int x;
}
int RetInt() {
	int y;
}
int aa() {
	if (RetBool()) {
		int x;
	}
	if (RetInt()) {
		int x;
	}
}



void main() { print("I hate this course"); }



