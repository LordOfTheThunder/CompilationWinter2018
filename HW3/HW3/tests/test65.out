---end scope---
x INT 0
---end scope---
y INT 0
---end scope---
x INT 0
---end scope---
---end scope---
x INT 0
---end scope---
line 11: type mismatch
